package.json

```diff
@@ -125,7 +125,9 @@
     "winston": "^3.1.0"
   },
   "devDependencies": {
-    "@storybook/react": "4.0.0-alpha.8",
+    "@babel/plugin-transform-modules-commonjs": "^7.2.0",
+    "@babel/plugin-transform-runtime": "^7.2.0",
```

Comment:
This is the dev version: https://babeljs.io/docs/en/next/babel-plugin-transform-runtime.html#installation